components/AntHiveBotVertical.vue

```diff
@@ -99,9 +99,6 @@ export default {
         return this.getStringTruncated(this.bot.displayName, 10)
       }
     },
-    getUserAvatar() {
-      return this.user && this.user.avatar
-    },
     getUsername() {
       return this.user && this.user.username
```

Comment:
if this.user && this.user.username{ 
return this.user.username
}else {
return "deleted"
}